docsets/JUCE/docset.json

```diff
@@ -11,6 +11,10 @@
         "Jules Utility Class Extensions"
     ],
     "specific_versions": [
+        {
+            "version": "7.0.2",
+            "archive": "versions/7.0.2/JUCE.tgz"
```

Comment:
This file does not exist. You added `versions/7.1.2/JUCE.tgz` instead

Reply:
ack! it was supposed to be 7.0.2 but i typo'd the branch name and used that. should be fixed now?